presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -545,6 +546,26 @@ public void testDropTableIfExists()
     }
 
     @Test
+    public void testOrderByChar()
+            throws Exception
+    {
+        assertUpdate("CREATE TABLE char_order_by (c_char char(2))");
+        assertUpdate("INSERT INTO char_order_by (c_char) VALUES(CAST('a' as CHAR(2))), (CAST('a\0' as CHAR(2)))", 2);
```

Comment:
Sure. It does not test comparison, though. It'd test truncation of longer than defined chars.